### PR TITLE
Update GAE plugin and ReadMes

### PR DESCRIPTION
**Pull request by averikitsch**

### CODE REVIEW

File: appengine-standard-java8/helloworld-gae-javasdk-tools/pom.xml [hunk 1]
```diff
@@ -124,13 +124,12 @@ limitations under the License.
...
-        <version>1.9.77</version>
+        <version>2.2.0</version>
         <configuration>
-          <!-- change to your app Id and version -->
-<!--
-          <appId>MY-APP-ID</appId>
-          <version>1</version>
- -->
+          <!-- set w/ -DprojectId=myProjectId on command line -->
+          <projectId>myProjectId</projectId>
```
Comment (lesv) [inline appengine-standard-java8/helloworld-gae-javasdk-tools/pom.xml]: `can be set` perhaps?

File: appengine-standard-java8/helloworld-gae-javasdk-tools/pom.xml [hunk 2]
```diff
@@ -124,13 +124,12 @@ limitations under the License.
...
+          <!-- "GCLOUD_CONFIG" allows for an autogenerated GAE version -->
+          <version>GCLOUD_CONFIG</version>
```
Comment (lesv) [inline appengine-standard-java8/helloworld-gae-javasdk-tools/pom.xml]: You have the choice of specifying your own versions, or letting Google autogenerate the version.

File: appengine-standard-java8/helloworld-gae-launch-dataflow/README.md [hunk 3]
```diff
@@ -58,6 +58,20 @@ detailed instructions.
...
+with your Google Cloud Project Id:
+
+```
+<plugin>
+  <groupId>com.google.cloud.tools</groupId>
+  <artifactId>appengine-maven-plugin</artifactId>
+  <version>2.2.0</version>
+  <configuration>
+    <projectId>myProjectId</projectId>
+    <version>GCLOUD_CONFIG</version>
```
Comment (lesv) [inline appengine-standard-java8/helloworld-gae-launch-dataflow/README.md]: You may wish to explain that "GCLOUD_CONFIG" is a special version that asks Google to auto-generate a version for you.

**Comment by averikitsch** [comment]
@lesv how does this look?

**Pull request merged**